public/javascripts/SVLabel/css/svl-modal.css

```diff
@@ -74,7 +74,7 @@
 }
 
 .modal-foreground p {
-    font-size: 12pt;
```

Comment:
As far as I see, the message is the only <p> tag. 12pt was pushing the button down too much, sometimes even out  of bounds.